packages/remix/src/build.ts

```diff
@@ -278,6 +278,7 @@ async function createRenderFunction(
     runtime: nodeVersion.runtime,
     shouldAddHelpers: false,
     shouldAddSourcemapSupport: false,
+    operationType: 'SSR',
```

Comment:
Whats the difference between API and SSR?

Reply:
I think the intent is that `SSR` returns HTML and `API` returns something else. These labels can be whatever makes the most sense for a given framework.

In that sense, "SSR" seemed like a reasonable default label for this. We could change it to a more Remix-specific value, but I'm not sure what that would be here. This is the main render function, right? Maybe we call this "Render".

@TooTallNate: you added this builder originally. What do you think makes sense here?

Reply:
SSR is appropriate in this case I think.

Reply:
Do we need to document those differences here or will it be in the BOA v3 docs?

The comment above is confusing to me since its not exhaustive:

```
// defaults to "API" (or "ISR" for prerenders)
```

Reply:
That's fair. In this repo, that comment doesn't make sense.

There will be a note about this in BOAv3, but this PR's code isn't about BOA.

How about we generically say this here:

```
// framework-specific lambda type label
```

[`958f1fb` (#8887)](https://github.com/vercel/vercel/pull/8887/commits/958f1fb70e6fac69ec7e3f2ca4933e599bb22432)

Reply:
Lets remove the comment. We should either add JSDoc for all of these or none of these properties.

Reply:
I don't agree that we should all-or-nothing these. Some information is better than none. I do agree that jsdoc'ing all of them would be good.

I can make this one a jsdoc for now. [`a0b30ee` (#8887)](https://github.com/vercel/vercel/pull/8887/commits/a0b30ee6fb4c3047d899468cff5e0f38926e20c1)